keyboards/delta/readme.md

```diff
@@ -0,0 +1,19 @@
+# Delta
+
+![Delta](https://i.imgur.com/fmrSAjb.jpg)
+
+[QuadrumLabs](http://quadrum.xyz/) Delta, PCB by [Lfgberg](https://github.com/lfgberg).
+
+[Album](https://www.imgur.com/a/WNj2h2a)
+
+To enter bootloader mode, either utilize the reset button near the MCU, or short the ESC key for bootmagic lite.
+
+* Keyboard Maintainer: [Lfgberg](https://github.com/lfgberg)
+* Hardware Supported: Delta PCB
+* Hardware Availability: [Vala Supply](http://vala.supply/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```